sample-project/settings.gradle.kts

```diff
@@ -1,8 +1,9 @@
-
 rootProject.name = "sample-project"
-
 //We want to stick to the legacy plugin
-System.setProperty("at.asitplus.gradle","legacy")
+//System.setProperty("at.asitplus.gradle","legacy")
```

Comment:
so, we don't want to stick to the legacy plugin? :wink:

Reply:
that was for testing. thanks for catching that!